tests/unittests/Foundation/NSJSONSerializationTests.m

```diff
@@ -117,11 +119,15 @@ void VerifyDataWithJSONObjectThrows(id testObject) {
     id testObject6 = @{ @"foo" : @"1", @"bar" : @"2" };
     id testObject7 = @{ @"foo" : @{ @"bar" : @"1" }, @"foo2" : @{ @"bar2" : @"2" } };
     id testObject8 = [NSUUID UUID];
+    id testObject9 = @{ @"key": @"value",
+                        @"nullkey":[NSNull null],
+                        @"array":@[@{@"subnullkey":[NSNull null]}]};
```

Comment:
Long story.  Didn't have commit hook setup in my new enlistment.  Good catch.

---
In reply to: [77276409](https://github.com/Microsoft/WinObjC/pull/953#discussion_r77276409) [](ancestors = 77276409)